types/generated-snapshot/2021-11-03/index.ts

```diff
@@ -2014,11 +2014,11 @@ export interface R2PutOptions {
   onlyIf?: R2Conditional | Headers;
   httpMetadata?: R2HTTPMetadata | Headers;
   customMetadata?: Record<string, string>;
-  md5?: ArrayBuffer | string;
-  sha1?: ArrayBuffer | string;
-  sha256?: ArrayBuffer | string;
-  sha384?: ArrayBuffer | string;
-  sha512?: ArrayBuffer | string;
+  md5?: (ArrayBuffer | ArrayBufferView) | string;
```

Comment:
Isn't this a breaking change?

Reply:
Nope. It already accepted both. The original type generation for kj::Array<kj::byte> is incorrect